playwright/fixtures/d-installer.yaml

```diff
@@ -0,0 +1,73 @@
+products:
```

Comment:
wondering if we want to name this folder 'fixtures', because playwright has their own, see: https://github.com/yast/d-installer/pull/399/files#diff-ea689e8e8cd1fb47d6827c3875ee50131c59166f93dbebe54d5d510fcdc8ee6d

Reply:
we can rename it, any suggestions?

Reply:
config makes more sense, doesn't collide with future folder 'fixtures' for playwright fixture. thanks!